src/odemis/gui/cont/streams.py

```diff
@@ -331,6 +335,20 @@ def _add_stream_setting_controls(self):
             conf = stream_config.get(vaname)
             self.add_setting_entry(vaname, va, hw_comp=None, conf=conf)
 
+    def _add_axis_controls(self):
+        """
+        Add controls for the axes that are connected to the stream
+        """
+        if hasattr(self.stream, "axis_map") and self.stream.axis_map:
+            # Add Axes (in same order as config)
+            axes_names = util.sorted_according_to(list(self.stream.axis_map.keys()), list(self._stream_config.keys()))
+            for axisname in axes_names:
+                real_ax_name, comp = self.stream.axis_map[axisname]
```

Comment:
You don't need real_ax_name, just the va_name, and you don't need the comp => drop this line.

Reply:
So now.. the display reads "Filter" rather than "band" since I didn't use the real name. Is that ok?

Reply:
That's the whole point. In the GUI, we don't care of the name on the hardware. The important name is the name on the stream.